docs/integrations/github/org.md

```diff
@@ -73,13 +73,15 @@ catalog:
       - id: github
         githubUrl: https://github.com
         orgs: ['organization-1', 'organization-2', 'organization-3']
+        alwaysUseDefaultNamespace: true
         schedule:
           initialDelay: { seconds: 30 }
           frequency: { hours: 1 }
           timeout: { minutes: 50 }
       - id: ghe
         githubUrl: https://ghe.mycompany.com
         orgs: ['internal-1', 'internal-2', 'internal-3']
+        alwaysUseDefaultNamespace: false
```

Comment:
Tbh I think we should avoid listing this as an option here in case people copy-paste it. With multiple orgs it's generally gonna make sense to have separate namespaces